README.md

```diff
@@ -30,6 +30,7 @@ changes:
       times. If `true`, all values will be collected in an array. If
       `false`, values for the option are last-wins. **Default:** `false`.
     * `short` {string} A single character alias for the option.
+    * `defaultValue` {string | boolean} The default option value when it is not set by args.
```

Comment:
Should this be `string | boolean | string[] | boolean[]`?

Reply:
Done